lib/Extractor/Candidates.cpp

```diff
@@ -35,6 +36,11 @@
 #include <unordered_set>
 #include <tuple>
 
+static llvm::cl::opt<bool> ExploitBPCs(
+    "souper-exploit-bpcs",
```

Comment:
How about souper-exploit-blockpcs? I think it doesn't hurt to use a complete name here.